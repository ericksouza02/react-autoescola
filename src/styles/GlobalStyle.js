import styled from "styled-components";
import { createGlobalStyle } from "styled-components";

const GlobalStyle = createGlobalStyle`
    @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
    body.sb-show-main.sb-main-padded{
    padding: 0;
    }

    html{
        font-family: 'Roboto', sans-serif;
        font-weight: 300;
        font-size: 100%;
        color: #fff;
        box-sizing: border-box;
    }

    *, *::before, *:after {
        box-sizing: inherit;
    }

    body {
        margin: 0;
        padding: 0;
    }
`

export default GlobalStyle;
