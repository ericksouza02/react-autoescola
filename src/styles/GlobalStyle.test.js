import { render } from "@testing-library/react";
import GlobalStyle from '../styles/GlobalStyle'

test('match snapshot GlobalStyle', () => {
    const {baseElement} = render(<GlobalStyle/>)

    expect(baseElement).toMatchSnapshot()
});

