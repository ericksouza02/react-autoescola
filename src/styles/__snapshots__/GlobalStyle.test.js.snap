// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot GlobalStyle 1`] = `
<body>
  <div />
</body>
`;
